src/css/components/player.scss

```
@@ -75,11 +75,17 @@
 
         .volume-controls {
             display: none;
-            
-            .icon-volume {
+
+            .icon-volume  svg {
                 width: 24px;
                 line-height: 0;
             }
+
+            .icon-mute svg {
+                width: 22px;
+                line-height: 0;
+                margin-right: 2px;
+            }
```

Y a encore un petit décalage pour moi, tu peux tester avec ça sinon :

```suggestion

            .icon-volume,
            .icon-mute {
                line-height: 0;

                svg {
                    height: 20px;
                }
            }

            .icon-mute svg {
                margin-right: 2.5px;
            }
```